.snippets/code/develop/parachain-devs/runtime-development/FRAME/pallet-testing/function-call-testing.rs

```diff
@@ -0,0 +1,18 @@
+#[test]
+fn it_works_for_valid_input() {
+    new_test_ext().execute_with(|| {
+        // Call an extrinsic or function
+        assert_ok!(TemplateModule::some_function(Origin::signed(1), valid_param));
```

Comment:
Maybe we could make a small note that because an account is `u64`, that this is why this works?

Reply:
Oh I believe that's explained in Line 41